analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/SymbolicValue.cs

```diff
@@ -22,11 +22,14 @@
 using System.Collections.Immutable;
 using System.Linq;
 using SonarAnalyzer.Helpers;
+using SonarAnalyzer.SymbolicExecution.Constraints;
 
 namespace SonarAnalyzer.SymbolicExecution.Roslyn
 {
     public sealed record SymbolicValue
     {
+        public static readonly SymbolicValue Null = new SymbolicValue().WithConstraint(ObjectConstraint.Null);
```

Comment:
Cache to save memory on something, that repeats often

Reply:
There will never be any current one. Literal operation is a leaf that cannot get constraints from external places (contrary to symbols from assignments for example).

Reply:
Closing this in favor of the discussion here https://github.com/SonarSource/sonar-dotnet/pull/5884/files#r928601061